ros/src/computing/perception/detection/packages/lidar_tracker/nodes/euclidean_cluster/euclidean_cluster.cpp

```diff
@@ -211,6 +229,18 @@ int main (int argc, char** argv)
 		ROS_INFO("euclidean_cluster > No points node received, defaulting to velodyne_points, you can use _points_node:=YOUR_TOPIC");
 		points_topic = "/points_raw";
 	}
+	publish_ground = false;
+	if (private_nh.getParam("publish_ground", publish_ground))
+	{
+		ROS_INFO("Publishing /points_ground point cloud...");
+		pub_filtered = h.advertise<sensor_msgs::PointCloud2>("/points_filtered",1);
```

Comment:
`ROS_INFO` says publishing **/points_Ground**, but this argument is **/points_filtered**.
Why ?